docs/adapters.rst

```diff
@@ -44,6 +44,14 @@ HttpAdapter
     api = Iota('https://service.iotasupport.com:14265')
     api = Iota(HttpAdapter('https://service.iotasupport.com:14265'))
 
+    # Use HTTPS with basic authentication and 60 seconds timeout:
+    api = Iota('https://USERNAME:MYIRIPASSWORD@service.iotasupport.com:14265',
+               timeout=60)
```

Comment:
❔ I'm not sure that this works — I believe the `timeout` argument only works with an `HttpAdapter` instance.

Reply:
Yes, that was a bit sloppy, copy-paste...